test/net/sourceforge/kolmafia/maximizer/MaximizerTest.java

```diff
@@ -25,30 +23,39 @@ public void init() {
 
   @Test
   public void changesGear() {
-    // 1 helmet turtle.
-    loadInventory("{\"3\": \"1\"}");
+    addItem("helmet turtle");
+    assertTrue(maximize("mus"));
+    assertEquals(1, modFor("Buffed Muscle"), 0.01);
```

Comment:
Probably doesn't need to change in this PR, but it's probably better to assert on expected gear rather than observed modifiers (which is more indirect and harder to understand).

(I recognize this is existing code. Up to you either way.)

Reply:
In general, I half agree, but I think there are cases where `modFor` is easier: e.g. weapon / off-hand that could be in either order, accessories that again could be in any order.

I think it's nicer to leave this as it is for this PR. I might change my mind after I think about this more / figure out a clean way of doing it.

Reply:
[both.gif](https://tenor.com/I0sW.gif)

Reply:
which is to say, testing modFor tests that the mod is applied, it's the expected amount, there are no other buffs from the maximizer, etc.

It doesn't test that the helmet turtle is selected, but that some item that makes the mod go to 1.  

Practically, that has to be the helmet turtle, because it's the only item in inventory, but these are two different things and could fail in different ways.  Which would both need to be checked the same way, so this is totally a nit...

Reply:
Yeah. If your goal is specifically to check the modifier (as is the case with the noncombat test), then I agree that it's better to assert on the modifier's value.

If your goal is to check that a particular item is or isn't equipped but don't care which slot it's in, that's a place for improving the testing ergonomics (e.g. adding `void assertRecommended(String itemName) { ... }` that's slot-agnostic).